langlib/lang.stream/src/main/ballerina/stream.bal

```diff
@@ -148,8 +186,13 @@ public isolated function iterator(stream<Type,CompletionType> stm) returns objec
 
 # Closes a stream.
 #
+# ```ballerina
+# stream<int> scoreStream = [45, 60, 75, 30, 90].toStream();
+# _ = scoreStream.close();
+# ```
+#
```

Comment:
Example needs to be after the function description, before the param descriptions.

Reply:
My bad. Fixed.